lib/home_page.dart

```diff
@@ -14,7 +20,34 @@ class _HomePageState extends State<HomePage> {
   Widget build(BuildContext context) {
     return Scaffold(
       appBar: AppBar(
-        title: Text(widget.title),
+        title: Text(GlobalLocalizations.of(context).chatApp),
+      ),
+      body: Column(
+        children: <Widget>[
+          MaterialButton(
+            child: Text(
+              'Polski',
```

Comment:
To też powinno być w locale data jako np. nazwa języka, a nie w kodzie D:

Reply:
ten home_page to tylko sprawdzałem czy działa, docelowo będzie przecież inny

Reply:
testy deweloperskie w kodzie nie powinny być wrzucane do mastera